src/platform/test.rs

```diff
@@ -452,9 +504,68 @@ fn cross_process() {
                (data, vec![], vec![]));
 }
 
+#[cfg(not(any(feature = "force-inprocess", target_os = "android")))]
+#[test]
+#[ignore]
+fn cross_process_sender_transfer_server()
```

Comment:
Because it's the good  kind of hacky, running with `--ignored` is fine -- these tests all pass by themselves, since they look for magic strings in the args :)